pkgs/top-level/python-packages.nix

```diff
@@ -15595,6 +15595,8 @@ in {
   pika-pool = callPackage ../development/python-modules/pika-pool { };
   platformio = callPackage ../development/python-modules/platformio { };
 
+  pykms = callPackage ../development/libraries/kmsxx { };
```

Comment:
We are already using ```pykms``` for something else although it's not under ```pythonPackages```.

Reply:
@peterhoeg ?
$ grep -Ir pykms | grep -v .git
pkgs/top-level/python-packages.nix:  pykms = callPackage ../development/libraries/kmsxx { };

Reply:
@peterhoeg Ok, seeing it after rebase. But that's another package. Need to do something with name.